charts/osdfir-infrastructure/charts/grr/values.yaml

```diff
@@ -45,9 +49,15 @@ fleetspeak:
     ## @param fleetspeak.frontend.image Sets the Fleetspeak fronend container image to use.
     ##
     image: "ghcr.io/google/fleetspeak:v0.1.17"
+    ## @param fleetspeak.frontend.address Sets the Fleetspeak frontend listen to use in a load balancer.
+    ##
+    address: ""
     ## @param fleetspeak.frontend.listenPort Sets the Fleetspeak frontend listen port to use.
     ##
-    listenPort: 4443
+    listenPort: 30443
+    ## @param fleetspeak.frontend.expose Defines how to expose the fleetspeak frontend [node|internal|external]
```

Comment:
If we could wrap the `internal` and `external` variables underneath something like `gcp.ingress.expose` then we could leave the default as `node` as currently is without having to specify it. In general for the other charts any gcp specific functionality is under the `gcp` variable